src/ios/CDVBarcodeScanner.mm

```diff
@@ -351,13 +352,16 @@ - (void)scanBarcode {
     self.viewController = [[CDVbcsViewController alloc] initWithProcessor: self alternateOverlay:self.alternateXib];
     // here we set the orientation delegate to the MainViewController of the app (orientation controlled in the Project Settings)
     self.viewController.orientationDelegate = self.plugin.viewController;
-
+    if (@available(iOS 13.0, *)) {
```

Comment:
would it work the same with the new iPad OS

Reply:
Yes. I will work.